notebooks/L1_working_with_age_ensembles.ipynb

```diff
@@ -25,9 +25,10 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def mapDataToEnsemble(chronValues, paleoValues, chronDepth, paleoDepth,
</pre><p>Does this require both chronValues and paleoValues to be present? It looks as though they are both essential with this syntax. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LinkedEarth/PyleoTutorials/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Yes, in this case `chronValues` refers to the time axis (or matrix), so its essential.